tests/pytorch/test_sanity.py

```diff
@@ -10,6 +10,7 @@
 import pytest
 import os
 
+import transformer_engine.pytorch
```

Comment:
Why is this needed?

Reply:
I find it convenient to be able to access a class without explicitly doing `from ... import ...`:
https://github.com/NVIDIA/TransformerEngine/blob/649b04c8a99c9fc2c432af3313030700fe0333f8/tests/pytorch/test_sanity.py#L1393

It's just a matter of style though. In the module we explicitly list the imports to order to guarantee only relative imports, but isn't relevant for tests since we always do absolute imports. Also, [Google's style guide recommends against it](https://google.github.io/styleguide/pyguide.html#22-imports).